config/config_test.go

```diff
@@ -996,6 +997,31 @@ var expectedConf = &Config{
 				},
 			},
 		},
+		{
+			JobName: "vultr",
+
+			HonorTimestamps: true,
+			ScrapeInterval:  model.Duration(15 * time.Second),
+			ScrapeTimeout:   DefaultGlobalConfig.ScrapeTimeout,
+
+			MetricsPath:      DefaultScrapeConfig.MetricsPath,
+			Scheme:           DefaultScrapeConfig.Scheme,
+			HTTPClientConfig: config.DefaultHTTPClientConfig,
+
+			ServiceDiscoveryConfigs: discovery.Configs{
+				&vultr.SDConfig{
+					HTTPClientConfig: config.HTTPClientConfig{
+						Authorization: &config.Authorization{
+							Type:        "Bearer",
+							Credentials: "abcdef",
+						},
+						FollowRedirects: true,
```

Comment:
```suggestion
						FollowRedirects: true,
						EnableHTTP2: true,
```

Reply:
ah thanks! I wasn't sure if this needed to be added or not